map/src/main/kotlin/edu/artic/map/MapFragment.kt

```diff
@@ -352,9 +348,10 @@ class MapFragment : BaseViewModelFragment<MapViewModel>() {
                 groundOverlayGenerated.filter { it }
         ) { floor, _ ->
             floor
-        }.subscribe {
-            buildingGroundOverlay.setImage(BitmapDescriptorFactory.fromAsset("AIC_Floor$it.png"))
-        }.disposedBy(disposeBag)
+        }.filter { floor -> floor > -1 }
```

Comment:
Please compare with Int.MIN_VALUE (the default used in ArticTour,etc.) instead.

Reply:
Updated.